v63004/u_inref/gtm8202.csh

```diff
@@ -0,0 +1,131 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+setenv gtm_test_jnl SETJNL
+setenv gtm_repl_instance mumps.repl
+setenv gtm_repl_instname INSTANCE1
+
+#setting this skips turning on replication in the passive start script
+setenv gtm_test_repl_skipsetreplic ""
+
+
+echo "#Create 3 database files for 3 regions, mumps.dat, a.dat, and b.dat for DEFAULT, AREG, and BREG respectively"
+$gtm_tst/com/dbcreate.csh mumps 3 255 8000 16384 1024 1024 1024 > db_log.txt
+
+$MUPIP set -repli=on  -reg "AREG" >>& db_log.txt
+$MUPIP set -repli=on  -reg "BREG" >>& db_log.txt
+$MUPIP set -repli=off -journal=disable -reg "DEFAULT" >>& db_log.txt
+
+$gtm_tst/com/passive_start_upd_enable.csh >>& passive_start.out
+
+echo "#calls gtm8202.m to set variables"
+$ydb_dist/mumps -run gtm8202 > gtm8202.m.log
+
+echo "#  single region extract -SEQNO=~0,1,~2 "
```

Comment:
In this case (and many other similar usages below), "#  single" should be "# Single"